sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -176,6 +176,12 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
       programYearTotalPartTimeCSGP: 4,
       programYearTotalPartTimeCSPT: 2,
       programYearTotalPartTimeSBSD: 6,
+      programYearTotalPartTimeBooksAndSuppliesCost: 0,
+      programYearTotalPartTimeFederalFSC: 0,
+      programYearTotalPartTimeProvincialFSC: 0,
+      programYearTotalPartTimeReturnTransportationCost: 0,
+      programYearTotalPartTimeScholarshipsBursaries: 0,
+      programYearTotalPartTimeSpouseContributionWeeks: 0,
```

Comment:
Can we mock some data into the workflow/assessment data columns to have some numbers here?